AspNet.Security.OAuth.Providers.sln

```diff
@@ -587,10 +589,6 @@ Global
 		{C90BA18B-E6C4-4E84-AFCC-58A4EE13BA40}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{C90BA18B-E6C4-4E84-AFCC-58A4EE13BA40}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{C90BA18B-E6C4-4E84-AFCC-58A4EE13BA40}.Release|Any CPU.Build.0 = Release|Any CPU
-		{815DA59A-E884-4BAD-B16C-D0B550B40A8D}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{815DA59A-E884-4BAD-B16C-D0B550B40A8D}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{815DA59A-E884-4BAD-B16C-D0B550B40A8D}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{815DA59A-E884-4BAD-B16C-D0B550B40A8D}.Release|Any CPU.Build.0 = Release|Any CPU
```

Comment:
Why have these lines been removed? I would expect more lines similar to these to have been added for a new provider.